docs/index.md

```diff
@@ -69,11 +69,11 @@ see the [tags on this repository](https://github.com/eu-digital-identity-wallet/
 
 See the list of [contributors](https://github.com/eu-digital-identity-wallet/architecture-and-reference-framework/graphs/contributors) who participated in this project.
 
-## License
+## Licence
 
 This project is licensed under the [Attribution 4.0
 International](http://creativecommons.org/licenses/by/4.0/) - see the
-[LICENSE](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/LICENSE) file for details.
+[LICENCE](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/LICENCE) file for details.
```

Comment:
This update introducing a broken link and a 404 error. We if we keep this change we should also change the file name in the target repository 

![image](https://github.com/user-attachments/assets/d3080be2-c581-4e06-842a-bcbfd7d33aa9)

Reply:
The Licence file name has been updated from LICENSE -> LICENCE